x/escrow/init_test.go

```diff
@@ -38,31 +37,32 @@ func TestGenesisKey(t *testing.T) {
   ]}`
 
 	var opts weave.Options
-	require.NoError(t, json.Unmarshal([]byte(genesis), &opts))
+	assert.IsErr(t, nil, json.Unmarshal([]byte(genesis), &opts))
```

Comment:
I think it's better to explicitely test for `nil` using `assert.Nil`

Reply:
I think it's nice to treat all the errors the same way. Mater of preference I guess